command/crypto/otp/verify.go

```diff
@@ -67,23 +73,73 @@ as "300ms", "-1.5h" or "2h45m". Valid time units are "ns", "us" (or "µs"), "ms"
 }
 
 func verifyAction(ctx *cli.Context) error {
-	args := ctx.Args()
-	secretFile := args.Get(0)
+	var (
+		secret, algStr string
+		period         uint
+		digits         int
+	)
 
+	secretFile := ctx.String("secret")
```

Comment:
You've changed how secret file is passed, can we keep both cases, and as this is a required parameter, can you return an `errs.RequiredFlag(..)` error.